lib/models/voie.js

```diff
@@ -4,6 +4,8 @@ const {getFilteredPayload} = require('../util/payload')
 const mongo = require('../util/mongo')
 const {validPayload} = require('../util/payload')
 const Numero = require('./numero')
+const nomAltSchema = require('./nom-alt')
+const {getNomAltDefault} = require('./nom-alt')
```

Comment:
```suggestion
const nomAltSchema, {getNomAltDefault) = require('./nom-alt')
```

ou alors juste import `nomAltSchema ` et faire `nomAltSchema. getNomAltDefault `